frontend/.eslintrc.cjs

```diff
@@ -115,6 +115,18 @@ module.exports = {
     "@typescript-eslint/no-unsafe-call": "off",
     "@typescript-eslint/no-unsafe-member-access": "off",
     "@typescript-eslint/no-unsafe-return": "off",
+    // "@typescript-eslint/strict-boolean-expressions": [
```

Comment:
leaving this in here since its good to turn on in development, but there are lots of noise that is not worth fixing in this PR